components/scheduler/templates/resource-grouping-header.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Resource Grouping Header
+page_title: Scheduler - Resource Grouping Header Template
+description: Use custom resource grouping header rendering through a template in the scheduler for Blazor.
+slug: scheduler-templates-resource-grouping-header
+tags: telerik,blazor,scheduler,templates,resource,grouping,header
+published: True
+position: 13
+---
+
+# Resource Grouping Header Templates
+
+You can use the `SchedulerResourceGroupHeaderTemplate` to customize the rendering of the Scheduler resource grouping header cells. This allows you to change the appearance of the content, add custom content or any HTML elements.
+
+The `SchedulerResourceGroupHeaderTemplate` is invoked for each resource when the Scheduler is configured to have resources and grouping. The template applies in both horizontal and vertical grouping.
+
+The `SchedulerResourceGroupHeaderTemplate` can be defined at the root level of the Scheduler and individually for each [Scheduler views]({%slug scheduler-views-overview%}). When configured at the root, the template applies to all views. If a `SchedulerResourceGroupHeaderTemplate` is defined at the view level, it will override the root-level template for that specific view.
```

Comment:
I recommend listing the two declaration options in a bullet list for easier readability. The details for each declaration can be listed next to the respective bullet.